pkg/mapper/configmap/mapper.go

```diff
@@ -52,6 +52,24 @@ func (m *ConfigMapMapper) Map(canonicalARN string) (*config.IdentityMapping, err
 		}, nil
 	}
 
+	ralm, err := m.RoleArnLikeMapping(canonicalARN)
+	if err == nil {
+		return &config.IdentityMapping{
+			IdentityARN: canonicalARN,
+			Username:    ralm.Username,
+			Groups:      ralm.Groups,
+		}, nil
+	}
+
+	ualm, err := m.UserArnLikeMapping(canonicalARN)
+	if err == nil {
+		return &config.IdentityMapping{
+			IdentityARN: canonicalARN,
+			Username:    ualm.Username,
+			Groups:      ualm.Groups,
+		}, nil
+	}
+
```

Comment:
This should not be necessary. the `m.RoleMapping()` and `m.UserMapping()` methods should simply check whether the `canonicalARN` variable matches any of the `RoleMapping` or `UserMapping` structs in the `mapStore`, regardless of whether the `RoleMapping` contains an ARN-specific string match or an ARNLike match.